index.bs

```diff
@@ -1873,15 +1873,16 @@ given:</p>
 </ol>
 
 <p class="note">A <a>WebVTT line cue setting</a> configures the offset of the cue box from the video
-viewport's edge in the direction opposite to the <a lt="WebVTT cue writing direction">writing
-direction</a>. For horizontal cues, this is the vertical offset from the top of the video viewport.
-The offset is for the <a lt="WebVTT cue line start alignment">start</a>, <a lt="WebVTT cue line
-center alignment">center</a>, or <a lt="WebVTT cue line end alignment">end</a> of the cue box,
-depending on the <a>WebVTT cue line alignment</a> value - <a lt="WebVTT cue line start
-alignment">start</a> by default. The offset can be given either as a percentage of the video
-dimension or as a line number. Line numbers are based on the size of the first line of the cue.
-Positive line numbers count from the start of the video viewport (the first line is numbered 0),
-negative line numbers from the end of the viewport (the last line is numbered &#x2212;1).</p>
+viewport's edge in the direction orthogonal to the <a lt="WebVTT cue writing direction">writing
+direction</a>. For horizontal cues, this is the vertical offset from the top of the video viewport,
+for vertical cues, it's the horizontal offset. The offset is for the <a lt="WebVTT cue line start
+alignment">start</a>, <a lt="WebVTT cue line center alignment">center</a>, or <a lt="WebVTT cue line
+end alignment">end</a> of the cue box, depending on the <a>WebVTT cue line alignment</a> value - <a
+lt="WebVTT cue line start alignment">start</a> by default. The offset can be given either as a
+percentage of the video dimension or as a line number. Line numbers are based on the size of the
```

Comment:
This is the size in the block progression direction, i.e. the vertical height for horizontal cues, right?

Reply:
yes.

Reply:
Yes.

Reply:
Thanks, I think the bit I was focusing on here was "percentage of the video dimension" which seems vague, since it's a two dimensional thing, and in general not square. Suggestion would be to qualify it by saying something like "percentage of the relevant writing-mode dependent video dimension"

Reply:
OK

Reply:
@nigelmegitt please see PR #420